cmd/hugo-golunr/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/riesinger/hugo-golunr/internal/post"
+	"github.com/spf13/afero"
+)
+
+var _fs = afero.NewOsFs()
+
+// baseURL should be parsed from the config.toml file in the hugo repo
+func main() {
+	flag.BoolVar(&post.Verbose, "v", false, "enable verbose output")
+	flag.BoolVar(&post.Verbose, "verbose", false, "enable verbose output")
+	flag.BoolVar(&post.WordSet, "w", false, "Convert content to set of words so only appear once")
+	flag.BoolVar(&post.WordSet, "wordset", false, "Convert content to set of words so only appear once")
+	flag.Parse()
+
+	fmt.Println("Version 1.6.0 2025-01-19")
```

Comment:
Printing the version every time the program runs is excessive. Keep the output to what's necessary.
Also, where does version 1.6.0 suddenly come from?